mava/networks/utils/sable/encoder_decoder_fns.py

```diff
@@ -0,0 +1,267 @@
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
```

Comment:
A high level suggestion here is that we have two files called `encode` and `decode` to keep similar to what we currently have for MAT.

Reply:
we can chat about this a bit because we may be able to merge some encoder decoder functions under one loop